docs/extending-highcharts/extending-highcharts.md

```diff
@@ -73,6 +71,25 @@ H.wrap(H.Series.prototype, 'drawGraph', function (proceed) {
 
 [Try it live](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/series/wrap-drawgraph/)
 
+Since v10, all module functions are available through a `window` event, `HighchartsModuleLoaded`. This includes utility functions and members that are not deliberately exposed. It is a powerful API, and was created with the intention of making it easier for the Highcharts developers and support team to provide temporary workarounds for bugs, or for special client requests. It should be warned that the module paths are not canonical, and may be subject to change as the product evolves. The `HighchartsModuleLoaded` event handlers must be defined before Highcharts is loaded. When loading ES modules, this will not work, and is also not necessary.
```

Comment:
> When loading ES modules, this will not work, and is also not necessary.

It would be nice to have this as a more detailed paragraph (under the try it link) showing the approach in ES modules.

Reply:
Also not sure about a lot of the commas in the first and third sentence. I personally would have written the sentences without any comma. If a native speaker took a look, I am fine with it.